inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -703,3 +703,29 @@ def increase_restart_count_or_error(obj, eng):
         save_workflow(obj, eng)
     else:
         raise WorkflowsError('Workflow restarted too many times')
+
+
+@with_debug_logging
+def find_all_interesting_affiliations(obj):
+    affiliations_mapping = current_app.config.get("AFFILIATIONS_TO_HIDDEN_COLLECTIONS_MAPPING", {})
+    affiliations = {affiliation for affiliations_list in get_value(obj.data, 'authors.raw_affiliations.value') for
+                    affiliation_raw in affiliations_list for affiliation in affiliation_raw.split()}
```

Comment:
can we do something simpler? Like intercept?

Reply:
WDYM?